copier/cli.py

```diff
@@ -189,6 +191,17 @@ def data_switch(self, values: List[str]) -> None:
             values: The list of values to apply.
                 Each value in the list is of the following form: `NAME=VALUE`.
         """
+        files = [e for e in values if "=" not in e]
+        values = [e for e in values if "=" in e]
+        print(files)
+        print(values)
+        for file in files:
+            if not Path(file).exists():
+                print(colors.red | f"File: {file} doesn't exist", file=sys.stderr)
+            if not re.match(r".+\.ya?ml$", file, re.IGNORECASE):
+                print(colors.red | f"File: {file} isn't a yaml file", file=sys.stderr)
+            d = load_template_config(Path(file), True)
+            self.data.update(d)
         for arg in values:
             key, value = arg.split("=", 1)
```

Comment:
I'd like it more if you used this same loop instead of creating a new one which re-defines the `values` argument. All would be more obvious to the reader. Besides, you can use this line to know if there's a `=` in the argument, as there'll be a `ValueError` otherwise.